docs/configuration/config-file-reference.md

```diff
@@ -345,7 +345,8 @@ ha_tracker:
 
   kvstore:
     # Backend storage to use for the ring. Supported values are: consul, etcd,
-    # inmemory, memberlist, multi.
+    # inmemory. Memberlist is not available for ha_tracker since Gossip is too
+    # slow for HA purposes.
```

Comment:
This doc is auto-generated via code. I pushed a commit to change it in the code and then re-generate the doc.